packages/solutions/module-tools/package.json

```diff
@@ -64,6 +65,7 @@
     "@types/fs-extra": "9.0.13",
     "@types/jest": "^27",
     "@types/node": "^14",
+    "detect-indent": "6.1.0",
```

Comment:
`detect-indent` should be a devDependencies

Reply:
it`s a devDependencies. Important information is hidden due to interface collapse. 😁

Reply:
soga...